ast.hpp

```diff
@@ -1326,10 +1331,10 @@ namespace Sass {
   // "flat" strings.
   ////////////////////////////////////////////////////////////////////////
   class String : public Expression {
-    ADD_PROPERTY(bool, needs_unquoting);
+    ADD_PROPERTY(bool, sass_fix_1291);
```

Comment:
The `needs_unquoting` flag will not be needed with the full refactoring. I my WIP this flag is needed to have the same behavior as ruby sass (https://github.com/sass/sass/issues/1291) which will be deprecated. Which means the `sass_fix_1291` can probably be removed once sass has it deprecated. But you're right, currently it doesn't make sense as `needs_unquoting` is still in the code. Please see comment below about `String_Constant` vs. `String_Quoted`. I would favor to ignore it for now.